### PR TITLE
Added set origination settings to sip peer

### PR DESCRIPTION
**Pull request by jmulford-bw**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1373,3 +1374,9 @@ catch (e) {
   console.log(e);
 }
 ```
+
+## Set SipPeer Origination Settings
+
+```js
+
```
Comment (jchavez443) [inline README.md:1381]: Was this purposely left blank?

File: test/xml.json [hunk 2]
```diff
@@ -65,5 +65,6 @@
...
-  "csrResponse": "<?xml version=\"1.0\" encoding=\"UTF-8\" standalone=\"yes\"?><CsrResponse><OrderId>218a295f-4f8a-4d1a-ba55-3e0aac6207cb</OrderId><Status>RECEIVED</Status></CsrResponse>"
+  "csrResponse": "<?xml version=\"1.0\" encoding=\"UTF-8\" standalone=\"yes\"?><CsrResponse><OrderId>218a295f-4f8a-4d1a-ba55-3e0aac6207cb</OrderId><Status>RECEIVED</Status></CsrResponse>",
```
Comment (jchavez443) [inline test/xml.json:68]: Are you using the `"` just to be consistent.  I find the JS ``` to be much easier to read and use.

**Review by jchavez443 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (jmulford-bw) [reply README.md:1382]: nope. i'm dumb and forgot to put an example in there. updated

File: test/xml.json (hunk 2 above)
Comment (jmulford-bw) [reply test/xml.json:68]: Yeahh I'm just being consistent

**Pull request merged**